Solutions/CiscoSEG/Data Connectors/Connector_Cisco_SEG_CEF.json

```diff
@@ -62,7 +62,7 @@
     "instructionSteps": [
         { 
             "title": "", 
-            "description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected [**CiscoSEGEvent**](https://aka.ms/sentinel-CiscoSEG-parser) which is deployed with the Azure Sentinel Solution.",
+            "description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://aka.ms/sentinel-CiscoSEG-parser) to create the Kusto Functions alias, **CiscoSEGEvent**",

```

Comment:
@vu-socprime  Could you revert this description change? The parser is to be deployed with the solution and this text update would cause confusion regarding that.